backend/app/usecase/authenticator/credential.go

```diff
@@ -0,0 +1,6 @@
+package authenticator
+
+// Credential encodes the identity of API consumer.
```

Comment:
```suggestion
// Credential represents the identity of API consumer.
```
"Encodes" makes it look like is a function

Reply:
@arberiii The JWT token is actually encoding app ID, key, and digital signature. Give this context, what are your thoughts on this?

Reply:
Then makes sense.